LinearAlgebra/basic_tests.py

```diff
@@ -0,0 +1,7 @@
+import numpy as np
+
+
+my_vector = np.array([1, 3, 7])
+my_matrix = np.matrix([[1, 2, 3], [4, 5, 6]])
+
+print(my_matrix.dot(my_vector))
```

Comment:
This red sign means there was not a newline at the end of the file. I think you can setup PyCharm to do this for you automatically.